### PR TITLE
Implement Login Page frontend tests

### PR DESCRIPTION
**Pull request by msabas1**

### CODE REVIEW

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LoginPage.java [hunk 1]
```diff
@@ -76,12 +96,11 @@ public LoginPage(WebDriver driver) {
         // Initialize form and add elements
         formLogin = new Form(driver);
 
-        formLogin.setInput(IN_USERNAME_ID, IN_USERNAME_NAME);
```
Comment (achuggi1) [inline cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LoginPage.java]: Medium impact: other PRs have shown the LoginPage.login method used, which directly makes use of the formLogin.
Suggestion: Bring back the formLogin.

**Review by jmensah365 — DISMISSED** [review]
Removal of the formLogin prevents other tests from passing on my branch

**Review by achuggi1 — CHANGES_REQUESTED** [review]
Removal of formLogin's inputs and submit button has the potential to affect other team members that use the LoginPage to log in.

File: cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LoginPage.java (hunk 1 above)
Comment (msabas1) [reply cucumber-selenium-tests/src/test/java/com/skillstorm/PageObjects/LoginPage.java]: formLogin is back after accidental removal and all login tests passing

**Review by jmensah365 — APPROVED** [review]

**Review by Bryar-Frank — APPROVED** [review]

**Review by Anthowu07 — APPROVED** [review]

**Review by achuggi1 — APPROVED** [review]
LGTM

**Pull request merged**